### PR TITLE
Theme asgardeo saml agent

### PR DESCRIPTION
**Pull request by kanchanagodage**
### Proposed changes in this pull request
Asgardeo SAML Agent themed
Screenshot (611)

### CODE REVIEW

File: io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/error.jsp [hunk 1]
```diff
@@ -23,42 +23,33 @@
...
-        <div class="element-padding">
-            <%=exception.getMessage()%>
+    <div class="ui two column centered grid">
+        <div class="column center aligned">
+            <img src="images/logo-dark.svg" class="logo-image">
+        </div>
+        <div class="container">
+            <div class="header-title">
+                <h1>
+                    Java Based SAML Authentication Sample <br> (OIDC - Authorization Code Grant)
```
Comment (vihanga-liyanage) [inline io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/error.jsp:40]:
```suggestion
                    Java Based SAML Authentication Sample
```

File: io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/home.jsp [hunk 2]
```diff
@@ -23,76 +23,56 @@
...
-            <%
-                }
-            %>
-        </div>
-        <div class="element-padding">
-            <table>
+        <div class="container">
+            <div class="header-title">
+                <h1>
+                    Java Based SAML Authentication Sample <br> (OIDC - Authorization Code Grant)
```
Comment (vihanga-liyanage) [inline io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/home.jsp:44]:
```suggestion
                    Java-Based SAML Authentication Sample
```

File: io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/index.html [hunk 3]
```diff
@@ -21,44 +21,39 @@
...
-                <input style="height: 30px; width: 60px" type="submit" value="log in">
+    <div class="ui two column centered grid">
+        <div class="column center aligned">
+            <img src="images/logo-dark.svg" class="logo-image">
+        </div>
+        <div class="container">
+            <div class="header-title">
+                <h1>
+                    Java Based SAML Authentication Sample<br>
+                    (OIDC - Authorization Code Grant)
```
Comment (vihanga-liyanage) [inline io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/index.html:36]:
```suggestion
                    Java-Based SAML Authentication Sample
```

File: io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/home.jsp [hunk 4]
```diff
@@ -23,76 +23,56 @@
...
+                    <dl class="details">
+                        <dt><%=entry.getKey()%></dt>
+                        <dt><%=entry.getValue()%></dt>
+                    </dl>
+                    <%
+                            }
+                        }
+                    %>
+                </div>
+                <button class="btn primary" onClick="location.href=logout?SAML2.HTTPBinding=HTTP-POST">Logout</button>
```
Comment (vihanga-liyanage) [inline io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/home.jsp:72]:
```suggestion
                <form action="logout" method="GET">
                    <div class="element-padding">
                        <button class="btn primary" onClick="location.href=logout?SAML2.HTTPBinding=HTTP-POST" type="submit">Logout</button>
                    </div>
                </form>
```

File: io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/home.jsp (hunk 2 above)
Comment (kanchanagodage) [reply io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/home.jsp]: Fixed with 6be6c50

File: io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/error.jsp (hunk 1 above)
Comment (kanchanagodage) [reply io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/error.jsp]: Fixed with 6be6c50

File: io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/index.html (hunk 3 above)
Comment (kanchanagodage) [reply io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/index.html]: Fixed with 6be6c50

File: io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/home.jsp (hunk 4 above)
Comment (kanchanagodage) [reply io.asgardeo.tomcat.saml.agent.sample/src/main/webapp/home.jsp]: Fixed with 6be6c50

**Review by vihanga-liyanage — APPROVED** [review]

**Comment by ChamathNS** [comment]
"(OIDC - Authorization Code Grant)" needs to be removed. @kanchanagodage

**Review by ChamathNS — APPROVED** [review]

**Pull request merged**